src/dmd/toctype.d

```diff
@@ -75,6 +75,7 @@ public:
     override void visit(TypeAArray t)
     {
         t.ctype = type_assoc_array(Type_toCtype(t.index), Type_toCtype(t.next));
+        t.ctype.Tident = t.toPrettyChars(true);
```

Comment:
Only do this if debug information is actually generated (global.params.symdebug)?

Reply:
I'm just following Dynamic Arrays here. Notice function above in that file. Trying to avoid breaking any existing behavior and follow the flow.

Reply:
I've seen that, too. It could be limited to debug info generation, too. toPrettyChar can be expensive for huge template types.

Reply:
It should be limited to debug information generation, if I got the code path correctly. `toPrettyChars` can be expensive indeed, bat that's just 100% match of Dynamic Array here, just to be sure.